### ISSUE TITLE
1.6 version support

### ISSUE THREAD

**Issue by KrusnikViers** [LunaMultiplayer/LunaMultiplayer#262]
20 December was released new version (1.6.*) of KSP client, "To Vee or not To Vee". Could we expect this version support, and how long will it take?

Also I do recommend to put specific supported versions instead of "latest" in the description, since it's really misleading. It took some time to understand that 1.6 it too new for the server, and 1.4 is to old (actually, I had to look for supported versions in code, because have not found them in docs).

**Comment by gavazquez** [comment]
Yep, Christmas holidays. Will do once back and I have free time as I have lot of work. Feel free to do a push request tough
